### PR TITLE
Migration azure and samples: sql, cosmosdb, container service

### PR DESCRIPTION
**Pull request by ChenTanyi**
Test not passed yet, I'll fix it together with CI-pipeline

### CODE REVIEW

File: azure-samples/pom.xml [hunk 1]
```diff
@@ -87,6 +87,16 @@
             <groupId>com.azure</groupId>
             <artifactId>azure-storage-blob</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-documentdb</artifactId>
```
Comment (weidongxu-microsoft) [inline azure-samples/pom.xml]: Please take a look whether you can use latest cosmos (not sure, azure-cosmos 4.0.0?).

File: azure-samples/pom.xml [hunk 2]
```diff
@@ -87,6 +87,16 @@
...
+            <version>1.9.4</version>
+        </dependency>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
```
Comment (weidongxu-microsoft) [inline azure-samples/pom.xml]: I don't think we should do this explicitly in track2 (it is ok if it from another lib, but then you will not need to write it here). Check your code and make refactor.

File: azure-samples/pom.xml (hunk 1 above)
Comment (ChenTanyi) [reply azure-samples/pom.xml]: done, version is 4.0.1-beta.1

File: azure-samples/pom.xml (hunk 2 above)
Comment (ChenTanyi) [reply azure-samples/pom.xml]: done

**Pull request merged**